kyo-zio/shared/src/main/scala/kyo/ZIOs.scala

```diff
@@ -55,47 +42,32 @@ object ZIOs:
       * @tparam A
       *   The result type
       */
-    inline def get[R: zio.Tag, E, A](v: ZIO[R, E, A])(using Tag[Env[R]], Frame): A < (Env[R] & ZIOs) =
+    inline def get[R: zio.Tag, E, A](v: ZIO[R, E, A])(using Tag[Env[R]], Frame): A < (Env[R] & Abort[E] & Async) =
         compiletime.error("ZIO environments are not supported yet. Please handle them before calling this method.")
 
-    /** Runs a Kyo effect within the ZIO runtime.
+    /** Interprets a Kyo computation to ZIO. Note that this method only accepts Abort[E] and Async pending effects. Plase handle any other
+      * effects before calling this method.
       *
       * @param v
       *   The Kyo effect to run
-      * @param frame
-      *   Implicit Frame
-      * @tparam E
-      *   The error type
-      * @tparam A
-      *   The result type
       * @return
-      *   A ZIO effect that will produce A or fail with E
+      *   A zio.ZIO that, when run, will execute the Kyo effect
       */
-    def run[E, A](v: => A < (Abort[E] & ZIOs))(using frame: Frame): ZIO[Any, E, A] =
+    def run[A: Flat, E](v: => A < (Abort[E] & Async))(using frame: Frame): ZIO[Any, E, A] =
```

Comment:
I believe these functions should accept a `using zio.Trace` to avoid polluting stack traces.